synapse/storage/databases/main/roommember.py

```diff
@@ -1424,6 +1425,17 @@ def get_sliding_sync_rooms_for_user_txn(
             get_sliding_sync_rooms_for_user_txn,
         )
 
+    async def have_finished_sliding_sync_background_jobs(self) -> bool:
+        """Return if its safe to use the sliding sync membership tables."""
```

Comment:
the nittiest of nits.

```suggestion
        """Return if it's safe to use the sliding sync membership tables."""
```